File: modulo1_logica/aula2_decisao.cpp
// Aula 2: Tomada de Decisão (If / Else)
// Objetivo: Ensinar o computador a tomar decisões com base em condições.

#include <iostream>

using namespace std;

int main() {
    cout << "--- AULA 2: TOMADA DE DECISAO ---" << endl;

    // Vamos reaproveitar a ideia da média da aula passada
    double media_final = 6.5;

    cout << "A media do aluno foi: " << media_final << endl;
    cout << "Verificando situacao do aluno...\n" << endl;

    // A estrutura IF (Se) e ELSE (Senão)
    if (media_final >= 7.0) {
        cout << "Resultado: APROVADO!" << endl;
        cout << "Parabens, voce foi muito bem." << endl;
    }
    else if (media_final >= 5.0) {
        // ELSE IF significa "Senão, Se"
        cout << "Resultado: RECUPERACAO!" << endl;
        cout << "Estude um pouco mais para a prova final." << endl;
    }
    else {
        cout << "Resultado: REPROVADO." << endl;
        cout << "Vamos tentar de novo ano que vem." << endl;
    }

    // ==========================================
    // EXERCÍCIO PARA OS ALUNOS NA SALA DE AULA:
    // Modifique o valor da variável 'media_final' para 8.0 e depois para 4.0.
    // Observe qual mensagem o programa imprime na tela em cada caso.
    // ==========================================

    return 0;
}

File: modulo1_logica/aula2_decisao.py
# Aula 2: Tomada de Decisão (If / Else)
# Objetivo: Ensinar o computador a tomar decisões com base em condições.

print("--- AULA 2: TOMADA DE DECISÃO ---")

# Vamos reaproveitar a ideia da média da aula passada
media_final = 6.5

print(f"A média do aluno foi: {media_final}")
print("Verificando situação do aluno...\n")

# A estrutura IF (Se) e ELSE (Senão)
if media_final >= 7.0:
    print("Resultado: APROVADO! 🎉")
    print("Parabéns, você foi muito bem.")
elif media_final >= 5.0:
    # ELIF significa "Else If" (Senão, Se)
    print("Resultado: RECUPERAÇÃO! 📚")
    print("Estude um pouco mais para a prova final.")
else:
    print("Resultado: REPROVADO. 😔")
    print("Vamos tentar de novo ano que vem.")

# ==========================================
# EXERCÍCIO PARA OS ALUNOS NA SALA DE AULA:
# Modifique o valor da variável 'media_final' para 8.0 e depois para 4.0.
# Observe qual mensagem o Python imprime na tela em cada caso.
# ==========================================